src/test/java/core/basesyntax/service/parser/FruitRecordDtoParserImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service.parser;
+
+import core.basesyntax.dto.FruitRecordDto;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.fileservice.FileReader;
+import core.basesyntax.service.fileservice.FileReaderForCsvImpl;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitRecordDtoParserImplTest {
+    private static final String VALID_CSV_FILE = "src/test/resources/valid_instructions.csv";
+    private static final String FIRST_INVALID_CSV_FILE =
+            "src/test/resources/invalid_instructions.csv";
+    private static final String SECOND_INVALID_CSV_FILE =
+            "src/test/resources/invalid_instructions_2.csv";
+    private static final String THIRD_INVALID_CSV_FILE =
+            "src/test/resources/invalid_instructions_3.csv";
+    private static final String FOURTH_INVALID_CSV_FILE =
+            "src/test/resources/invalid_instructions_4.csv";
+    private static final String EMPTY_CSV_FILE = "src/test/resources/empty.csv";
+    private static final String INVALID_PATH = "src/test/resourcesempty.csv";
+    private static final FruitRecordDtoParser PARSER = new FruitRecordDtoParserImpl();
+    private static final FileReader FILE_READER = new FileReaderForCsvImpl();
```

Comment:
```suggestion
    private static final FileReader fileReader = new FileReaderForCsvImpl();
```
if class is not deeply immutable we don't use such naming 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names